MPower/app/build.gradle

```diff
@@ -11,9 +11,6 @@ android {
         versionName "1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
         multiDexEnabled true
-        buildConfigField 'String', 'STUDY_ID', studyId
-        buildConfigField 'String', 'STUDY_NAME', studyName
-        buildConfigField 'String', 'BRIDGE_URL', bridgeUrl
         buildConfigField 'String', 'CERTIFICATE_NAME', certificateName
```

Comment:
This still exists for now because the new UploadHandler isn't fully integrated, and so the old code path is still active.